src/components/WalletCompare.js

```diff
@@ -114,50 +114,50 @@ const walletFeatures = [
   {
     id: "has_card_deposits",
     emoji: ":credit_card:",
-    title: <Translation id="page-find-wallet-buy-card" />,
-    description: <Translation id="page-find-wallet-buy-card-desc" />,
+    title: "page-find-wallet-buy-card",
+    description: "page-find-wallet-buy-card-desc",
   },
   {
     id: "has_explore_dapps",
     emoji: ":world_map:",
-    title: <Translation id="page-find-wallet-explore-dapps" />,
-    description: <Translation id="page-find-wallet-explore-dapps-desc" />,
+    title: "page-find-wallet-explore-dapps",
+    description: "page-find-wallet-explore-dapps-desc",
   },
   {
     id: "has_defi_integrations",
     emoji: ":money_with_wings:",
-    title: <Translation id="page-find-wallet-fi-tools" />,
-    description: <Translation id="page-find-wallet-fi-tools-desc" />,
+    title: "page-find-wallet-fi-tools",
+    description: "page-find-wallet-fi-tools-desc",
   },
   {
     id: "has_bank_withdrawals",
     emoji: ":bank:",
-    title: <Translation id="page-find-wallet-withdraw" />,
-    description: <Translation id="page-find-wallet-withdraw-desc" />,
+    title: "page-find-wallet-withdraw",
+    description: "page-find-wallet-withdraw-desc",
   },
   {
     id: "has_limits_protection",
     emoji: ":shield:",
-    title: <Translation id="page-find-wallet-limits" />,
-    description: <Translation id="page-find-wallet-limits-desc" />,
+    title: "page-find-wallet-limits",
+    description: "page-find-wallet-limits-desc",
   },
   {
     id: "has_high_volume_purchases",
     emoji: ":whale:",
-    title: <Translation id="page-find-wallet-volume" />,
-    description: <Translation id="page-find-wallet-voluem-desc" />,
+    title: "page-find-wallet-volume",
+    description: "page-find-wallet-voluem-desc",
   },
   {
     id: "has_dex_integrations",
     emoji: ":repeat:",
-    title: <Translation id="page-find-wallet-swaps" />,
-    description: <Translation id="page-find-wallet-swaps-desc" />,
+    title: "page-find-wallet-swaps",
+    description: "page-find-wallet-swaps-desc",
   },
   {
     id: "has_multisig",
     emoji: ":busts_in_silhouette:",
-    title: <Translation id="page-find-wallet-multisig" />,
-    description: <Translation id="page-find-wallet-multisig-desc" />,
+    title: "page-find-wallet-multisig",
+    description: "page-find-wallet-multisig-desc",
```

Comment:
@HursitTarcan I left only the keys here so that we translate this strings in the `WalletCompare`. This way you receive a string prop and not a component/element.